packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.ts

```diff
@@ -51,6 +51,14 @@ export const useAccordionHeader = (
   const size = useContextSelector(AccordionContext, ctx => ctx.size);
   const id = useId('accordion-header-', props.id);
   const panel = useDescendants(accordionItemDescendantContext)[1] as AccordionItemDescendant | undefined;
+  const handleKeyDown = React.useCallback(
+    (ev: React.KeyboardEvent) => {
+      if (ev.key === ' ' || ev.key === 'Enter') {
```

Comment:
I don't think this is that important and it's your call, but natively when you keyboard click it still actually triggers mouse click. Please take a look at `useTriggerElement.ts` for an example.

Reply:
Let's also use `@fluentui/keyboard-key` for consistency, you can find examples in `react-menu` package.